src/securejoin/qrinvite.rs

```diff
@@ -95,7 +111,22 @@ impl TryFrom<Qr> for QrInvite {
                 invitenumber,
                 authcode,
             }),
-            _ => bail!("Unsupported QR type"),
+            Qr::AskJoinBroadcast {
+                broadcast_name,
+                grpid,
+                contact_id,
+                fingerprint,
+                authcode,
+                invitenumber,
+            } => Ok(QrInvite::Broadcast {
+                broadcast_name,
+                grpid,
+                contact_id,
+                fingerprint,
+                authcode,
+                invitenumber,
+            }),
+            _ => bail!("Unsupported QR type: {qr:?}"),
```

Comment:
Not sure that logging QRs which contain secrets is good. Then they may be scanned from e.g. bug reports